client/src/Components/Common/UplodeImage/uploadImage.js

```diff
@@ -0,0 +1,88 @@
+import React, { Fragment, useState } from 'react'
+import AddIcon from '@material-ui/icons/Add'
+// import RaisedButton from 'material-ui/core/RaisedButton';
+import { Button } from '@material-ui/core'
+import IconButton from '@material-ui/core/IconButton'
+import axios from 'axios'
+import useStyles from './uploadImageStyle'
+import {Box} from '@material-ui/core'
+import {Grid} from '@material-ui/core'
+
+const UploadImage = ({getImageUrl}) => {
+  const classes = useStyles()
+  const [image, addImage] = useState('')
+  const [imageName, addImageName] = useState('choose file')
+  const [uploadedImage, setUploadedImage] = useState('')
+
+  const handaleImage = async (e) => {
```

Comment:
Small typo:
```suggestion
  const handleImage = async (e) => {
```

**note:** also update spelling where it's called below


**thought:** I think it's pretty obvious in this context, but sometimes single letter variables can get confusing especially `e` since it's used for both `event` and `error`